spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -29,6 +33,33 @@
         get :show, id: -1
       }.to raise_error(ActionController::RoutingError)
     end
+
+    it 'returns correct ban if user muted' do
+      create :ban, :mute, user: @user
+      get :show, id: @user.id
+      expect(response).to be_success
+      expect(json['bans']['mute']).to eq(true)
+    end
+    it 'returns correct ban if user gather banned' do
+      create :ban, :gather, user: @user
+      get :show, id: @user.id
+      expect(response).to be_success
+      expect(json['bans']['gather']).to eq(true)
+    end
+    it 'returns correct ban if user site banned' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.